tests/snmp/test_snmp_queue_counters.py

```diff
@@ -48,10 +48,28 @@ def test_snmp_queue_counters(duthosts,
         = "docker exec snmp snmpwalk -v2c -c {} {} {}".format(
             creds_all_duts[duthost.hostname]['snmp_rocommunity'], hostip,
             Ethernet0_queue_cntrs_oid)
+    # Generate sonic-cfggen commands for multi-asic and single-asic duts
+    nslist = duthost.get_asic_namespace_list()
+    if duthost.sonichost.is_multi_asic and nslist is not None:
+        cmd = "sonic-cfggen -n asic0 -d --print-data > {}".format(ORIG_CFG_DB)
```

Comment:
Instead of getting the configuration only of asic0 always, should we run the test for randomly selected one asic_index using enum_rand_one_asic_index

Reply:
Modified the test to select random asic_index based on enum_rand_one_asic_index. Please check.